src/main/java/org/spongepowered/api/item/recipe/ShapedRecipe.java

```diff
@@ -52,90 +61,47 @@
     int getHeight();
 
     /**
-     * Gets the ingredient required in the given coordinates.
+     * Gets the ingredient required by the given symbol.
      *
-     * @param x The x coordinate
-     * @param y The y coordinate
+     * @param symbol The ingredient symbol
      * @return The ingredient
      */
-    Optional<ItemStack> getIngredient(int x, int y);
-
-    /**
-     * Gets the ingredient required in the given position.
-     *
-     * @param pos The position
-     * @return The ingredient
-     */
-    Optional<ItemStack> getIngredient(Vector2i pos);
+    Optional<ItemStack> getIngredient(char symbol);
```

Comment:
I'm not sure on the usefulness of this method because it requires knowledge of how the recipe was created in the builder. I think an `(x, y)` coord would be better suited here.